var/spack/repos/builtin/packages/py-pacifica-uploader/package.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPacificaUploader(PythonPackage):
+    """Python Pacifica Uploader Library"""
+
+    homepage = "https://github.com/pacifica/pacifica-python-uploader/"
+
+    version('0.3.1', sha256='adda18b28f01f0b1e6fbaf927fec9b8cf07c86f1b74185bed2a624e8a4597578', url="https://pypi.io/packages/source/p/pacifica-uploader/pacifica-uploader-0.3.1.tar.gz")
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-wheel', type='build')
```

Comment:
Dep not needed, missing dep on requests

Reply:
this is needed it seems, and i'm adding requests.

Reply:
This is only needed to build wheels, I don't think it's needed to build from source